src/main/resources/permissions/system-user-permissions.csv

```diff
@@ -1 +1,2 @@
 inventory.instances.collection.get
+inventory.contributor-types.collection.get
```

Comment:
Why does you use this permission? From what I see the request is made to inventory-storage. So, I think we need to use: "inventory-storage.contributor-types.collection.get" - https://s3.amazonaws.com/foliodocs/api/mod-inventory-storage/p/contributor-type.html

Reply:
Agree, fixed.